Sources/DeepLearning/Operators/NN.swift

```diff
@@ -86,6 +86,8 @@ public extension Tensor where Scalar: TensorFlowFloatingPoint {
 /// A padding scheme. Used by padding, convolution, and pooling ops.
 // @_frozen // SR-9739
 public enum Padding {
+    /// The "explicit" padding scheme.
+    case explicit(paddings: [Int32])
```

Comment:
Remove the `paddings:` label since 1) it's merely repeating the type name, and 2) the plurality is fully conveyed by type information (array).